content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Announcing the general availability of the Command provider"
+
+date: 2024-06-20T11:02:20+02:00
+
+draft: false
+
+meta_desc: >-
+    Version v1 of the Pulumi Command provider marks the provider’s general availability (GA).
+    The release expands support for copying assets between local and remote hosts.
+
+meta_image: meta.png
+
+authors:
+    - thomas-kappler
+
+# None of the tags in BLOGGING.md seem to match, but `providers` exists already
+tags:
+    - providers
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
+
+# Here are some examples of posts we have made in the past for inspiration:
+# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
+# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
+# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
+# https://twitter.com/PulumiCorp/status/1763265391042654623
+# https://twitter.com/PulumiCorp/status/1762900472489185492
+# https://twitter.com/PulumiCorp/status/1755637618631405655
+
+social:
+    twitter:
+    linkedin:
+---
+
+Today, we’re happy to announce version v1.0.0 of the Pulumi Command provider. This release marks the provider’s official transition from preview status to general availability (GA).
+Since its first preview release in late 2021, many users found the flexibility of the Command package valuable. In fact, it’s currently our tenth-most popular provider, so it's time to make things official!
```

Comment:
```suggestion
Since its first preview release in late 2021, thousands of Pulumi users incorporated the Command package into their cloud infrastructure projects to help manage local and remote command execution and filesystem operations. In fact, the Command provider is already our tenth-most popular provider, so it's time to make things official!
```

FWIW - I see 5256 users who have used it in Pulumi Cloud, and actually that it is the 9th most popular :-).